wrap.go

```diff
@@ -63,9 +63,27 @@ func fsevtCallback(stream C.FSEventStreamRef, info uintptr, numEvents C.size_t,
 			Flags: EventFlags(flags[i]),
 			ID:    uint64(ids[i]),
 		}
+		//IF the event has a renameFlag, exclude the RenameFlag and pass the rest (may include other operations)
+		//Cache the RenameEvent
+		if events[i].Flags&ItemRenamed == ItemRenamed {
+			cacheRenameEvent := events[i]
+			events[i].Flags = events[i].Flags ^ ItemRenamed
+			cacheRenameEvent.Flags = ItemRenamed
+			if events[i].Flags&ItemIsFile == ItemIsFile {
```

Comment:
Doesn't the cached rename event already include these flags? Why do we have to set them again?

Reply:
No because I set the cached event flag to be ItemRename and then I check to see if the file system type is known then I add that to the cached event flag as well